### ISSUE TITLE
Log Reports: Offline status issue (1.2.0beta2)

### ISSUE THREAD

**Issue by nholland94** [MinaProtocol/mina#9210]
We have observed an increase in nodes going into the "Offline" status with the latest `1.2.0beta1` release. The new `1.2.0beta2` release contains new logs that we believe will help us confirm and identify the root cause of the issue.

We ask that users running the new `1.2.0beta2` release who encounter this issue to assist us by attaching logs to this issue. In order for the logs to be most helpful to us, we ask that users who encounter the issue mark the timestamp of when it occurred, and let the node run for a few minutes after the issue first occurs before recording the logs (so that we can see the behavior of the node in the period of time following the issue).

**Comment by lampardlamps** [comment]
sorry could you provide more details on how to record the logs, is it still using `export-logs`? If I encounter it and let it run a few more minutes, then isn't the timestamp of when that happens a few minutes ago?

**Comment by nholland94** [comment]
> sorry could you provide more details on how to record the logs, is it still using `export-logs`? If I encounter it and let it run a few minutes, then isn't the timestamp a few minutes ago?

Yes, `export-logs` is still the preferred way to bundle up logs for us to view. The timestamp the issue occurs at helps us more accurately pinpoint when the problem began, if you are able to provide it. A "few minutes ago" isn't an accurate enough time frame for us to pinpoint the precise time where the issue started happening (and we don't have a log message that will tell us when it began, just some statistics to look at after the issue occurs).

**Comment by Manaudou** [comment]
> > sorry could you provide more details on how to record the logs, is it still using `export-logs`? If I encounter it and let it run a few minutes, then isn't the timestamp a few minutes ago?
> 
> Yes, `export-logs` is still the preferred way to bundle up logs for us to view. The timestamp the issue occurs at helps us more accurately pinpoint when the problem began, if you are able to provide it. A "few minutes ago" isn't an accurate enough time frame for us to pinpoint the precise time where the issue started happening (and we don't have a log message that will tell us when it began, just some statistics to look at after the issue occurs).

Stuck at old block heights. - Block 44193 
25 minutes in status unchanged. -Sync status: Synsed 
After 9 blocks, I saved the logs.
File size 221mb

**Comment by jrwashburn** [comment]
My mainnet node started going offline again on 7/23 05:43:41, and then was offline at least 35 more times until my node reset 7/24 09:37 (via monitor script) because it was too far off (max observed vs. max observed invalidated hit a delta of 13.) It has remained in sync sync then.

This folder https://drive.google.com/drive/folders/15zftGXX8gV-T-RA3uoL20kNuXh3NCv8Y?usp=sharing contains tar files from mina client export-logs every 4 hours during the 24 hours from before the first occurrence on  7/23 for the following 24 hours.

Additionally, it contains mina-mainnet-vt-1_2021-07-23_05-43-first offline.zip which includes the output from the monitoring script, which shows the return status from mina client status every five minutes, and the specific 2 log files surrounding the first offline occurrence (those files were extracted from mina-mainnet-vt-1_2021-07-23_12-00-58.tar.)

**Comment by lampardlamps** [comment]
I've just the offline issue on 1.2.0beta7 within 2hrs of local uptime:

`Global number of accounts:                     59434
Block height:                                  66448
Max observed block height:                     66448
Max observed unvalidated block height:         66448
Local uptime:                                  1h58m47s
Ledger Merkle root:                            jxpVNB8FebH5FGarh1SbLiZRFDtAwnSH5FdSdvas4bcrmXFNUaK
Protocol state hash:                           3NKxNUJtjhAKFTUv1xnd2qXf2LLwjgM2xXfzoGFTLtaVY2NV7nUV
Chain id:                                      5f704cc0c82e0ed70e873f0893d7e06f148524e3f0bdae2afb02e7819a0c24d1
Git SHA-1:                                     9b5bbaa17a77a15b559400dec578236f782655bc
Configuration directory:                       /home/ubuntu/.mina-config
Peers:                                         59
User_commands sent:                            0
SNARK worker:                                  None
SNARK work fee:                                100000000
Sync status:                                   Offline`

logs are here: https://1drv.ms/u/s!AlOGZ7BQqAI1iKluW0MQGfc6zBNfDA?e=ChFdH4

**Comment by dkijania** [comment]
I'm leaning to closing this issue. version is pretty old and it looks like issue more connected with infra/version rather that general issue we can fix in newest code
